types/src/shared/utils/config.ts

```diff
@@ -16,4 +16,14 @@ export class EnvironmentConfig {
 
     return cachedValue;
   }
+
+  static getOptionalEnvVariable(key: string): string | undefined {
+    if (!this.cache[key]) {
```

Comment:
If the variable isn't defined, we will always enter this path, but I guess it's fine.

Reply:
Yes (IMO this whole caching thing is useless)